convert/jenkinsjson/json/pager_duty.go

```diff
@@ -0,0 +1,76 @@
+package json
+
+import (
+	"encoding/json"
+	"log"
+
+	harness "github.com/drone/spec/dist/go"
+)
+
+// ConvertPagerduty creates a Harness step for nunit plugin.
+func ConvertPagerDuty(node Node, arguments map[string]interface{}) *harness.Step {
+	incidentSource, _ := arguments["incidentSource"].(string)
+	resolve, _ := arguments["resolve"].(bool)
+	dedupKey, _ := arguments["dedupKey"].(string)
+	incidentSummary, _ := arguments["incidentSummary"].(string)
+	incidentSeverity, _ := arguments["incidentSeverity"].(string)
+	routingKey, _ := arguments["routingKey"].(string)
+
+	convertPagerduty := &harness.Step{
+		Id:   SanitizeForId(node.SpanName, node.SpanId),
+		Name: "Pagerduty",
+		Type: "plugin",
+		Spec: &harness.StepPlugin{
+			Image: "plugins/pagerduty",
+			With: map[string]interface{}{
+				"log_level":         "info",
+				"routing_key":       routingKey,
```

Comment:
```suggestion
				"routing_key":       <+input>,
```

changed it to get user input as routing key is a secret